### ISSUE TITLE
I have some problem

### ISSUE THREAD

**Issue by pzwstudy** [FuchenUSTC/VideoStudio#1]
Interesting work !
I have two problem 
1.This is a  entity_prompt file
![image](https://github.com/user-attachments/assets/c7337e16-1f02-4a08-bb85-659395dc2f00)
l want to make a image with this prompt, I should use which t2i model to generate this image ？
2. can you offer this long video pipeline soon?

**Comment by FuchenUSTC** [comment]
@pzwstudy Yes, you need to make the reference image via t2i models, e.g., SD-2.1 or SDXL. We are preparing the long video pipeline and will realse it soon.
